fastlane/Fastfile

```diff
@@ -59,6 +59,94 @@ lane :github_release do |options|
   )
 end
 
+desc "Update dependencies to latest GitHub releases"
+lane :update_native_dependencies_to_latest do |options|
+  latest_ios_release_info = get_latest_github_release(url: "RevenueCat/purchases-ios")
+  latest_android_release_info = get_latest_github_release(url: "RevenueCat/purchases-android")
+
+  latest_ios_release_name = latest_ios_release_info["name"]
+  latest_android_release_name = latest_android_release_info["name"]
+
+  UI.message("ℹ️  Latest RevenueCat iOS version: #{latest_ios_release_name}")
+  UI.message("ℹ️  Latest RevenueCat Android version: #{latest_android_release_name}")
+
+  update_ios_native_versions(version: latest_ios_release_name)
+  update_android_native_versions(version: latest_android_release_name)
+
+  run_pod_install
+end
+
+desc "Update purchases-ios dependency version"
+lane :update_ios_native_versions do |options|
+  dry_run = options[:dry_run]
+  if options[:version]
+    new_version_number = options[:version]
+  else
+    UI.user_error!("Missing `version` argument")
+  end
+
+  previous_version_number = parse_previous_ios_native_version
+
+  # Add to options so if we're chaining lanes together, the other lanes get it
+  options[:previous_ios_version] = previous_version_number
```

Comment:
I wonder if this (and the android lane version) are really needed... Do you foresee it being necessary for something?

Reply:
Removing. This actually doesn't work and I ended up doing it in a different way in https://github.com/RevenueCat/purchases-hybrid-common/pull/179 (which is based on top of this PR)